app.js

```diff
@@ -1,18 +1,18 @@
 const express = require("express")
 const { getTopics } = require("./controllers/nc-news-controllers")
 const app = express()
+const endpoints = require("./endpoints.json")
 
+app.get("/api", (request, response) => {
```

Comment:
Data shouldn't be imported into the app, so we need to extract this callback function as a controller 👀 If the data is imported into the app, it exists on the memory of the server all the time, even if a request isn't made to the endpoint that responds with it. This is inefficient for the performance of our server.

Reply:
Make sense, will do!!